packages/tenant-process/src/services/tenantService.ts

```diff
@@ -753,22 +762,15 @@ async function reevaluateTenantKind({
 }: {
   tenant: Tenant;
   readModelService: ReadModelService;
-}): Promise<{ updatedTenant: Tenant; tenantKindHasBeenUpdated: boolean }> {
+}): Promise<{ tenantKind: TenantKind; tenantKindHasBeenUpdated: boolean }> {
```

Comment:
I agree with you, but that means this function is not needed because what it does is just call the `getTenantKindLoadingCertifiedAttributes` function.
We missed the generality of the verification, but it is acceptable for this specific case.

Reply:
So if we brought it back to how it was before, making it return to the tuple `[updatedtenant, event]` again and called it `updatedTenantwithKindEvent`
I don't have much imagination with names 😆

Reply:
It's not necessary, just drop the function, use directly `getTenantKindLoadingCertifiedAttributes` and then check the eventual variation of the tenant kind, wherever is needed.